fiscalyear.py

```diff
@@ -603,6 +607,211 @@ def __ge__(self, other):
+        :rtype: FiscalMonth
+        """
+        today = FiscalDate.today()
+        return cls(today.fiscal_year, today.fiscal_month)
+
+    def __repr__(self):
+        """Convert to formal string, for repr().
+
+        >>> fm = FiscalMonth(2017, 1)
+        >>> repr(fm)
+        'FiscalMonth(2017,1)'
+        """
+        return '%s(%d, %d)' % (self.__class__.__name__,
+                               self._fiscal_year,
+                               self._fiscal_month)
+
+    def __str__(self):
+        """Convert to informal string, for str().
+
+        >>> fm = FiscalMonth(2017, 1)
+        >>> str(fy)
+        'FY2017 FM1'
+        """
+        return 'FY%d FM%d' % (self._fiscal_year, self._fiscal_month)
+
+    # TODO: Implement __format__ so that you can print
+    # fiscal year as 17 or 2017 (%y or %Y)
+
+    def __contains__(self, item):
+        """Returns True if item in self, else False.
+
+        :param item: The item to check
+        :type item: FiscalYear, FiscalQuarter, FiscalDateTime,
+            datetime, FiscalDate, or date
+        :rtype: bool
+        """
+        if isinstance(item, FiscalMonth):
+            return self == item
+        elif isinstance(item, datetime.datetime):
+            return self.start <= item <= self.end
+        elif isinstance(item, datetime.date):
+            return self.start.date() <= item <= self.end.date()
+        else:
+            raise TypeError("can't compare '%s' to '%s'" % (
+                type(self).__name__, type(item).__name__))
+
+    # Read-only field accessors
+
+    @property
+    def fiscal_year(self):
+        """:returns: The fiscal year
+        :rtype: int
+        """
+        return self._fiscal_year
+
+    @property
+    def fiscal_month(self):
+        """:returns: The fiscal month
+        :rtype: int
+        """
+        return self._fiscal_month
+
+    @property
+    def start(self):
+        """:returns: Start of the fiscal month
+        :rtype: FiscalDateTime
+        """
+
+        month = ((self._fiscal_month - START_MONTH) % 12 + 6) % 12 + 1
+
+        if month >= START_MONTH:
+            year = self._fiscal_year - 1
+        else:
+            year = self._fiscal_year
+
+        return FiscalDateTime(year, month, START_DAY)
+
+    @property
+    def end(self):
+        """:returns: End of the fiscal year
+        :rtype: FiscalDateTime
+        """
+        # Find the start of the next fiscal quarter
+        next_start = self.next_fiscal_month.start
+
+        # Substract 1 second
+        end = next_start - datetime.timedelta(seconds=1)
+
+        return FiscalDateTime(end.year, end.month, end.day,
+                              end.hour, end.minute, end.second,
+                              end.microsecond, end.tzinfo)
+
+    @property
+    def year(self):
+        """:returns: The fiscal year for the month
+        :rtype: FiscalYear
+        """
+        return FiscalYear(self._fiscal_year)
```

Comment:
I'm concerned about this attribute. If `fm.fiscal_year` returns an int and `fm.year` returns a `FiscalYear`  think a lot of people will be confused. I would remove this property for now as it's easy to create with `FiscalYear(fm.fiscal_year)`

Reply:
Yeah it confused the heck out of me when I was working on FiscalDay. It's used elsewhere in the code, but easy enough to replace as you say.